### ISSUE TITLE
Eclipse's test attribute

### ISSUE THREAD

**Issue by ben-manes** [melix/jmh-gradle-plugin#157]
Eclipse Photon introduced a `Contains test source` attribute for containerization of those classes. Recent Gradle / Buildship added support for this, which results in test sources not being visible to JMH sources in Eclipse. The workaround is to also make the JMH sources part of the test container.

```gradle
eclipse.classpath.file.whenMerged {
  entries.find { it.path == 'src/jmh/java' }.entryAttributes['test'] = 'true'
}
```

**Comment by sbrannen** [comment]
We've run into the same issue in the build for the Spring Framework, and in order to properly support JMH resources as well, we've had to modify the suggested workaround as follows.

```groovy
eclipse.classpath.file.whenMerged {
	entries.findAll { it.path =~ /src\/jmh\/(java|resources)/ }.each {
		it.entryAttributes['test'] = 'true'
	}
}
```

**Comment by ben-manes** [comment]
verified this was fixed in the newly released 0.7.2

**Comment by sbrannen** [comment]
I don't see a "milestone" or "release" for 0.7.2.

@melix, can you please add those?

Thanks in advance!
